docs/guide/time.rst

```diff
@@ -32,24 +66,64 @@ Here are few examples of formats which `sunpy.time.parse_time()` accepts::
     >>> parse_time('20070504_210812')
     <Time object: scale='utc' format='isot' value=2007-05-04T21:08:12.000>
 
-Each of the above returns the same `~astropy.time.Time` object ``<Time object: scale='utc' format='isot' value=2007-05-04T21:08:12.000>``. One of the most standard time formats used in solar
-physics is the number of seconds since 1979 January 01 called ``utime``. The parse_time
-function also accepts this as input, e.g.::
+Each of the above returns the same `~astropy.time.Time` object ``<Time object: scale='utc' format='isot' value=2007-05-04T21:08:12.000>``.
+
+One of the most common time formats used in unix is the number of seconds since 1979 January 01
```

Comment:
nope, it's not a unix timestamp, that's 1970 not 1979. This is weird and obscure.